tests/vereya/common.py

```diff
@@ -9,7 +9,7 @@
 from base_test import BaseTest
 
 
-def init_mission(mc, start_x, start_y, seed, forceReset="false", forceReuse="false"):
+def init_mission(mc, start_x, start_y, seed, forceReset="false", forceReuse="false", start_z=78):
```

Comment:
start_y should go with default value 78. In tests we mostly set different x and z coordinates

Reply:
@noskill  done.